### PR TITLE
Simple Table 508 Compliance Warning

### PR DESCRIPTION
**Pull request by jaygiang**
issue #369 

- Adds `area-label` to `<Icon>` component

### CODE REVIEW

File: packages/react-radfish/table/index.jsx [hunk 1]
```diff
@@ -183,14 +187,14 @@ const RADFishTable = ({
...
-            <Icon.FirstPage />
+            <Icon.FirstPage aria-label="Go to first page" />
           </Button>
           <Button
             onClick={() => handlePageChange(pageIndex - 1)}
             disabled={pageIndex === 0}
             data-testid="previous-page"
           >
-            <Icon.ArrowBack />
+            <Icon.ArrowBack aria-label="Go back a page" />
```
Comment (thgaskell) [inline packages/react-radfish/table/index.jsx:197]:
```suggestion
            <Icon.ArrowBack aria-label="Go to previous page" />
```

File: packages/react-radfish/table/index.jsx [hunk 2]
```diff
@@ -200,14 +204,14 @@ const RADFishTable = ({
             disabled={pageIndex >= totalPages - 1}
             data-testid="next-page"
           >
-            <Icon.ArrowForward />
+            <Icon.ArrowForward aria-label="Go forward a page" />
```
Comment (thgaskell) [inline packages/react-radfish/table/index.jsx:207]:
```suggestion
            <Icon.ArrowForward aria-label="Go to next page" />
```

**Review by thgaskell — DISMISSED** [review]
Thanks, looks good!

Offered a couple copy suggestions to keep the wording consistent.

File: packages/react-radfish/table/index.jsx (hunk 2 above)
Comment (jaygiang) [reply packages/react-radfish/table/index.jsx]: Thanks @thgaskell! Changes committed. Will need to review again.

**Review by thgaskell — APPROVED** [review]

**Pull request merged**